### PR TITLE
install a modern version of createrepo_c

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: puppet/modules/web/manifests/vhost/yum.pp [hunk 1]
```diff
@@ -61,6 +61,17 @@
     package { 'createrepo':
       ensure => present,
     }
+
+    yumrepo { 'pulpcore-3.16':
```
Comment (ekohl) [inline puppet/modules/web/manifests/vhost/yum.pp:65]: Should we scope this to EL7 or is it also needed on EL8?

File: puppet/modules/web/manifests/vhost/yum.pp [hunk 2]
```diff
@@ -61,6 +61,17 @@
     package { 'createrepo':
```
Comment (ekohl) [inline puppet/modules/web/manifests/vhost/yum.pp:61]: Is this still needed if we install `createrepo_c`?

Comment (evgeni) [reply puppet/modules/web/manifests/vhost/yum.pp:61]: probably not, but I wanted to keep a fallback, just to be sure ;)

File: puppet/modules/web/manifests/vhost/yum.pp (hunk 1 above)
Comment (evgeni) [reply puppet/modules/web/manifests/vhost/yum.pp:65]: no, el8 has a sufficiently new version, so can be el7 only

Comment (ekohl) [reply puppet/modules/web/manifests/vhost/yum.pp:65]: I just merged https://github.com/theforeman/foreman-infra/pull/1720 so let's try to keep it ready for EL8 :)

**Review by ekohl — APPROVED** [review]

**Pull request merged**